package.json

```diff
@@ -7,8 +7,9 @@
   "repository": "https://github.com/alphagov/paas-admin",
   "scripts": {
     "build": "npm run -s clean && webpack --config config/webpack.config.js",
-    "test": "(npm run -s test:unit -- --cov && npm run -s lint) && echo '\n✓ ALL TESTS PASSED!'",
-    "test:unit": "ENABLE_TESTS=true npm run -s build && tap --100 --no-coverage --nyc-arg='--instrument=false' --nyc-arg='--all' dist/*.test.js",
+    "watch": "ENABLE_WATCH=true ENABLE_TESTS=true npm run -s build",
```

Comment:
Shouldn't this command also do `webpack --watch`?

Reply:
we are not using the `--watch` arg ... we are setting `watch: true` in the webpack config to because we need to do more than just enable watch mode, we need to alter how tests are run (should not explode and bail out on failure when watching) and alter how the server runs (should hot reload modules when watching). We're not using _any_ of the webpack cli args it's all done through the config.